scripts/tar_web_vault.sh

```diff
@@ -18,13 +18,21 @@ mkdir -pv "${BASEDIR}/../${OUTPUT_FOLDER}"
 pushd "${VAULT_FOLDER}/apps/web"
 
 VAULT_VERSION=$(get_web_vault_version)
-OUTPUT_NAME="${OUTPUT_FOLDER}/bw_web_${VAULT_VERSION}"
+OUTPUT_NAME="${BASEDIR}/../${OUTPUT_FOLDER}/bw_web_${VAULT_VERSION}"
+DATE_FORMAT="${DATE_FORMAT:-%Y-%m-%dT%H:%M:%S%z}"
 
 mv build web-vault
 # Tar the web-vault
-tar -czvf "${BASEDIR}/../${OUTPUT_NAME}.tar.gz" web-vault --owner=0 --group=0
+tar -czvf "${OUTPUT_NAME}.tar.gz" web-vault --owner=0 --group=0
+
+# Preserve previous output directory
+if [[ -e ${OUTPUT_NAME} ]];
+then
+    mv ${OUTPUT_NAME}{,-$(date +${DATE_FORMAT} -r ${OUTPUT_NAME})}
```

Comment:
First, the arguments need to be quoted to prevent argument globing, else path's with spaces could cause issues.
Second, i think creating an archive of the previous output would be better, and after that remove it.
Also since `mv` has the same quirks as `cp` which is why you crated this PR in the first place.
This will only happen if you run this script twice or more right after each other, but it could happen.

```suggestion
    tar -czf "${OUTPUT_NAME}-$(date +"${DATE_FORMAT}" -r "${OUTPUT_NAME}").tar.gz" "${OUTPUT_NAME}" --owner=0 --group=0
    rm -rf "${OUTPUT_NAME}"
```

Reply:
Thanks for the review. Since we already create an archive, I think we can preserve that file instead and remove the directory.

>  This will only happen if you run this script twice or more right after each other, but it could happen.

Yeah, it happens frequently to me while testing changes.